src/cmdline.rs

```diff
@@ -315,10 +315,10 @@ pub struct InstallConfig {
     #[structopt(next_line_help(true))]
     pub network_dir: DefaultedString<NetworkDir>,
     /// Save partitions with this label glob
+    #[serde(skip_serializing_if = "is_default")]
     #[structopt(long, value_name = "lx")]
     // Allow argument multiple times, but one value each.  Allow "a,b" in
     // one argument.
-    #[serde(skip_serializing_if = "is_default")]
```

Comment:
OOC, was this actually causing compile issues?

Reply:
Not at all, it's just a cleanup.